openslides/utils/utils.py

```diff
@@ -1,25 +1,10 @@
 import roman
-from django.contrib.auth.models import Permission
-
-
-def delete_default_permissions(**kwargs):
-    """
-    Deletes the permissions, django creates by default for the admin.
-    """
-    # TODO: Find a way not to create the permissions in the first place.
-    #       Meta.default_permissions does not work, because django will
-    #       nevertheless create permissions for its own models like "group"
-    for p in Permission.objects.all():
-        if (p.codename.startswith('add') or
-                p.codename.startswith('delete') or
-                p.codename.startswith('change')):
-            p.delete()
```

Comment:
This code is not used at the moment. Someone has to reimplement a new solution in the users app. See #1643